proto/v1/vulnerability.proto

```diff
@@ -73,25 +73,35 @@ message VulnerabilityNote {
     // Required. The package this vulnerability affects.
     string affected_package = 5;
 
-    // Required. The minimum version of the package this vulnerability affects.
-    grafeas.v1.Version min_affected_version = 6;
-
-    // The [CPE URI](https://cpe.mitre.org/specification/) this vulnerability
-    // was fixed in. It is possible for this to be different from the
-    // affected_cpe_uri.
-    string fixed_cpe_uri = 7;
-
-    // The package this vulnerability was fixed in. It is possible for this to
-    // be different from the affected_package.
-    string fixed_package = 8;
-
-    // Required. The version of the package this vulnerability was fixed in.
-    // Setting this to VersionKind.MAXIMUM means no fix is yet available.
-    grafeas.v1.Version fixed_version = 9;
+    // Required. The version number at the start of an interval in which this vulnerability
+    // exists. A vulnerability can affect a package between version numbers
+    // that are disjoint sets of intervals (Example: [1.0.0-1.1.0], [2.4.6-2.4.8]
+    // and [4.5.6-4.6.8]).
```

Comment:
Please add: "each of which will be represented in its own Detail"

Reply:
ditto for the `affected_version_end` comment

Reply:
Addressed in PR #318